lib/client.js

```diff
@@ -474,6 +496,12 @@ function Client(opts) {
     call.json = true
     call.qs = call.qs || {};
 
+    call.qsStringifyOptions = {
+      arrayFormat: 'repeat'
+    };
```

Comment:
Yeah, was just curious. I know it shouldn't be 'indices' because we ran into an issue with that.